cicada/cli.py

```diff
@@ -201,9 +195,22 @@ def spread_schedules():
             required=True,
             help="List of target server_ids to spread schedules to",
         )
+        parser.add_argument(
+            "--commit",
+            default=False,
+            action="store_true",
+            help="Commits changes to backend DB, otherwise only print output",
+        )
+        parser.add_argument(
+            "--force",
+            default=False,
+            action="store_true",
+            help="If schedule is moving servers and also currently running, perform abort_running and adhoc_execute "
+            "| Only available when --commit is specified",
+        )
         # now that we're inside a subcommand, ignore the first TWO args
         args = parser.parse_args(sys.argv[2:])
-        spread_schedules.main(args)
+        spread_schedules.main(args.__dict__)
```

Comment:
it's not a good practice to call the dunder methods (`__xyz__`) directly, try `dict(args)`